docker-compose.yml

```diff
@@ -5,11 +5,12 @@ services:
     build:
       context: ./docker
       dockerfile: Dockerfile
-    image: myparcelcom/php:7.3-cli
+    image: myparcelcom/json-api:v1-dev
     working_dir: /opt/json-api
     volumes:
       - .:/opt/json-api
       - composercache:/tmp
+    user: ${CURRENT_UID}
```

Comment:
Just wondering, can we do the same with the auth server where we have permission issues with the .key files?

Reply:
Yes! We can actually do this in every project! This makes the container run as the current user (you) and we no longer have to fix the permissions using those shell functions in the `mp.sh` script!
I got this idea from Sam over the weekend!